### PR TITLE
Added 3 Nodejs books english version

### PR DESCRIPTION
**Pull request by Dirac156**
- Web development with Node JS and Express
- NodeJS Design Patterns
- Node For embedded system

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by shrn01 — COMMENTED** [review]
PDFDrive links are not supported. Please provide official links to resources

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1275,13 +1275,16 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Introduction to Node.js](https://nodejs.dev/learn) (HTML)
 * [Mastering Node](https://github.com/visionmedia/masteringnode) - visionmedia ([PDF](https://github.com/visionmedia/masteringnode/blob/master/book.pdf))
 * [Mixu's Node Book](http://book.mixu.net/node/)
-* [Node Documentation](https://nodejs.org/en/docs/)
+* [Node Design Patterns](https://ia801309.us.archive.org/5/items/HandbookOfNeuralComputingApplicationsPDFStormRG/Node.js%20Design%20Patterns%20-%20Casciaro,%20Mario%20%5BPDF%5D%5BStormRG%5D.pdf) - Mario Casciaro (PDF)
+* [Node Documentation](https://nodejs.org/en/docs/) (PDF)
+* [Node For Embedded System](https://www.pdfdrive.com/nodejs-for-embedded-systems-using-web-technologies-to-build-connected-devices-e157056110.html) - Pactrick Mulder and Kelsey Breseman (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1280]:
```suggestion
```
In general, we do not link to file sharing sites, or file hosting sites such as google docs. or dropbox.

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1275,13 +1275,15 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
-* [Node Documentation](https://nodejs.org/en/docs/)
+* [Node Design Patterns](https://ia801309.us.archive.org/5/items/HandbookOfNeuralComputingApplicationsPDFStormRG/Node.js%20Design%20Patterns%20-%20Casciaro,%20Mario%20%5BPDF%5D%5BStormRG%5D.pdf) - Mario Casciaro (PDF)
+* [Node Documentation](https://nodejs.org/en/docs/) (PDF)
 * [Node: Up and Running](https://www.oreilly.com/library/view/node-up-and/9781449332235/) - Tom Hughes-Croucher
 * [Node.js Notes for Professionals](http://goalkicker.com/NodeJSBook) - Compiled from StackOverflow Documentation (PDF)
 * [Node.js Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/nodejs) (PDF, Kindle) (email address *requested*, not required)
 * [Practical Node.js: Building Real-World Scalable Web Apps](https://github.com/azat-co/practicalnode) - Azat Mardan
 * [The Node Beginner Book](http://nodebeginner.org)
 * [The Node.js Handbook](https://flaviocopes.com/page/node-handbook/) - Flavio Copes (PDF, EPUB, Kindle) *(email address requested)*
+* [Web development with Node JS and Express](https://www.vanmeegern.de/fileadmin/user_upload/PDF/Web_Development_with_Node_Express.pdf) - Ethen Brown (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1286]:
```suggestion
This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.
```

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -1275,13 +1275,15 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
-* [Node Documentation](https://nodejs.org/en/docs/)
+* [Node Design Patterns](https://ia801309.us.archive.org/5/items/HandbookOfNeuralComputingApplicationsPDFStormRG/Node.js%20Design%20Patterns%20-%20Casciaro,%20Mario%20%5BPDF%5D%5BStormRG%5D.pdf) - Mario Casciaro (PDF)
+* [Node Documentation](https://nodejs.org/en/docs/) (PDF)
 * [Node: Up and Running](https://www.oreilly.com/library/view/node-up-and/9781449332235/) - Tom Hughes-Croucher
 * [Node.js Notes for Professionals](http://goalkicker.com/NodeJSBook) - Compiled from StackOverflow Documentation (PDF)
 * [Node.js Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/nodejs) (PDF, Kindle) (email address *requested*, not required)
 * [Practical Node.js: Building Real-World Scalable Web Apps](https://github.com/azat-co/practicalnode) - Azat Mardan
 * [The Node Beginner Book](http://nodebeginner.org)
 * [The Node.js Handbook](https://flaviocopes.com/page/node-handbook/) - Flavio Copes (PDF, EPUB, Kindle) *(email address requested)*
+This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1286]:
```suggestion
```

File: books/free-programming-books-langs.md [hunk 4]
```diff
@@ -1275,7 +1275,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Introduction to Node.js](https://nodejs.dev/learn) (HTML)
 * [Mastering Node](https://github.com/visionmedia/masteringnode) - visionmedia ([PDF](https://github.com/visionmedia/masteringnode/blob/master/book.pdf))
 * [Mixu's Node Book](http://book.mixu.net/node/)
-* [Node Documentation](https://nodejs.org/en/docs/)
+* [Node Design Patterns](https://ia801309.us.archive.org/5/items/HandbookOfNeuralComputingApplicationsPDFStormRG/Node.js%20Design%20Patterns%20-%20Casciaro,%20Mario%20%5BPDF%5D%5BStormRG%5D.pdf) - Mario Casciaro (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1278]:
```suggestion
* [Node.js Design Patterns](https://ia801309.us.archive.org/5/items/HandbookOfNeuralComputingApplicationsPDFStormRG/Node.js%20Design%20Patterns%20-%20Casciaro,%20Mario%20%5BPDF%5D%5BStormRG%5D.pdf) - Mario Casciaro (PDF)
```

**Pull request merged**